services/src/pro/quota/mod.rs

```diff
@@ -170,6 +182,41 @@ pub fn initialize_quota_tracking<U: UserDb + 'static>(
     QuotaTrackingFactory::new(quota_sender)
 }
 
+#[derive(Debug, Serialize, Deserialize, ToSchema, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct OperatorQuota {
+    pub operator_name: String,
+    pub operator_path: String,
```

Comment:
is this cloned a lot? maybe `Arc<str>`

Reply:
no, it's used for API output and filled from the database